packaging/os/zypper_repository.py

```diff
@@ -59,12 +59,14 @@
         required: false
         default: "no"
         choices: [ "yes", "no" ]
-    refresh:
+    autorefresh:
         description:
             - Enable autorefresh of the repository.
         required: false
         default: "yes"
         choices: [ "yes", "no" ]
+        aliases: [ "refresh" ]
+        version_added: "2.2"
```

Comment:
this was already there before 2.2, no need to add version_added, build will fail, but this is "ok" in this case